backend/eventd/eventd_test.go

```diff
@@ -69,6 +69,7 @@ func TestEventHandling(t *testing.T) {
 
 	// Make sure the event has been marked with the proper state
 	assert.Equal(t, types.EventPassingState, event.Check.State)
+	assert.Equal(t, badEvent.Timestamp, event.LastOK)
```

Comment:
We'll want to assert that event.Check.LastOK is equal to event.Timestamp.

Publishing a bad event seems out of place in this test, but I can look into that another time. Line 63 is where we're publishing our event. We want to make sure that eventd is correctly updating the LastOK field with the current timestamp.